app/furniture/marketplace/order.rb

```diff
@@ -1,6 +1,7 @@
 class Marketplace
   class Order < Record
     self.table_name = "marketplace_orders"
+    default_scope { where.not(status: :pre_checkout) }
```

Comment:
Why do we need a default scope here? How certain are we that this won't introduce yet another bug somewhere where an edge case isn't tested? (I'm not certain at all, personally.)

I'd rather we explicitly applied named scopes as needed, instead of having a default scope.

Reply:
Yea, this is a side effect of not having split the `marketplace_carts` and `marketplace_orders` tables out. I like putting it in a named scope!